File: grpc/src/main/java/client/grpc_client.py
import grpc
import user_pb2
import user_pb2_grpc

def run():
    channel = grpc.insecure_channel('localhost:9099')
    stub = user_pb2_grpc.userStub(channel)
    login_request = user_pb2.LoginRequest(username='user',password='user')
    login_response = stub.login(login_request)

    print(login_response)

if __name__ == "__main__":
    run()

File: grpc/src/main/java/client/GrpcClient.java
package client;

import com.example.User;
import com.example.userGrpc;
import io.grpc.ManagedChannel;
import io.grpc.ManagedChannelBuilder;

public class GrpcClient {

    public static void main(String[] args) {

        ManagedChannel channel = ManagedChannelBuilder.forAddress("localhost", 9099).usePlaintext().build();
        //stubs
        userGrpc.userBlockingStub userStub = userGrpc.newBlockingStub(channel);
        User.LoginRequest loginRequest = User.LoginRequest.newBuilder().setUsername("a").setPassword("a").build();
        User.APIResponse login = userStub.login(loginRequest);
        System.out.println(login.getResponseCode());


    }

}
